src/tdlib/user.rs

```diff
@@ -221,6 +221,13 @@ impl User {
         self.connect_notify_local(Some("avatar"), f)
     }
 
+    pub(crate) fn connect_user_type_changed_notify<F: Fn(&Self, &glib::ParamSpec) + 'static>(
+        &self,
+        f: F,
+    ) -> glib::SignalHandlerId {
+        self.connect_notify_local(Some("type"), f)
+    }
+
```

Comment:
Do we want to notify the message that the sender delete account?  It might be expensive to compute without major benefits

Reply:
Expensive meaning the user just deleted an account and we have to update 100 MessageRows

Reply:
I think we need it, but not in this way because:

1. It seems this method is just a redundant wrapper, the caller could directly the `connect_notify_local` function. We already have one, but I don't want to continue to use such trivial wrappers as they bloat the code.
2. This PR may be a goog opportunity to get rid of the expressions in `avatar.rs`. Since the `item` here can be more than one type, you could use `SignalGroups` like in the sidebar row. So you can bind the `notify_type` signal to one of the two groups.